.github/workflows/close-issue-with-banned-phrases.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Check for blocked phrases in issue title
         run: |
           ISSUE_TITLE=$(jq -r '.issue.title' "$GITHUB_EVENT_PATH")
-          BLOCKED_PHRASES=("twitter" "whatsapp" "logo" "google" "tiktok" "facebook" "slack" "discord" "bluesky" "spotify" "behance" "pix")
+          BLOCKED_PHRASES=("twitter" "whatsapp" "logo" "google" "tiktok" "facebook" "slack" "discord" "bluesky" "spotify" "behance" "pix" "x.com")
```

Comment:
There seems to be a linting issue with this, we'll need to add x.com to https://github.com/lucide-icons/lucide/blob/main/.cspell/custom-words.txt